src/test/compile-fail/E0101.rs

```diff
@@ -9,5 +9,7 @@
 // except according to those terms.
 
 fn main() {
-    let x = |_| {}; //~ ERROR E0101
+    let x = |_| {};
+    //~^ ERROR E0101
+    //~| cannot resolve type of expression
```

Comment:
Can you make both of these `//~| NOTE cannot resolve type of expression` instead?